x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -119,39 +124,18 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
                   'This index pattern does not contain any numeric type fields. The analytics job may not be able to come up with any outliers.',
               })
             }
-            isInvalid={!sourceIndexNameEmpty && (!sourceIndexNameValid || !sourceIndexNameExists)}
+            isInvalid={!sourceIndexNameEmpty && !sourceIndexNameValid}
             error={
-              (!sourceIndexNameEmpty &&
-                !sourceIndexNameValid && [
-                  <Fragment>
-                    {i18n.translate('xpack.ml.dataframe.analytics.create.sourceIndexInvalidError', {
-                      defaultMessage: 'Invalid source index name.',
-                    })}
-                    <br />
-                    <EuiLink
-                      href={`https://www.elastic.co/guide/en/elasticsearch/reference/${metadata.branch}/indices-create-index.html#indices-create-index`}
-                      target="_blank"
-                    >
-                      {i18n.translate(
-                        'xpack.ml.dataframe.stepDetailsForm.sourceIndexInvalidErrorLink',
-                        {
-                          defaultMessage: 'Learn more about index name limitations.',
-                        }
-                      )}
-                    </EuiLink>
-                  </Fragment>,
-                ]) ||
-              (!sourceIndexNameEmpty &&
-                !sourceIndexNameExists && [
-                  <Fragment>
-                    {i18n.translate(
-                      'xpack.ml.dataframe.analytics.create.sourceIndexDoesNotExistError',
-                      {
-                        defaultMessage: 'An index with this name does not exist.',
-                      }
-                    )}
-                  </Fragment>,
-                ])
+              !sourceIndexNameEmpty &&
+              !sourceIndexNameValid && [
+                <Fragment>
```

Comment:
As above, this should never be shown as you can only select a valid KIbana index pattern as the source index.